File: SEP3-TIER2/src/Server/Model/Project.java
package Server.Model;

import java.util.List;

public class Project {

    private int id;
    private String title;
    private String description;
    private List<Channel> channels;
    private String ownerusername;
    private List<ProjectUser> users;

    public Project(int id, String title, String description, List<Channel> channels, String ownerusername) {
        this.id = id;
        this.title = title;
        this.description = description;
        this.channels = channels;
        this.ownerusername = ownerusername;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public List<Channel> getChannels() {
        return channels;
    }

    public void setChannels(List<Channel> channels) {
        this.channels = channels;
    }

    public String getOwnerusername() {
        return ownerusername;
    }

    public void setOwnerusername(String ownerusername) {
        this.ownerusername = ownerusername;
    }

    public List<ProjectUser> getUsers() {
        return users;
    }

    public void setUsers(List<ProjectUser> users) {
        this.users = users;
    }

    @Override
    public String toString() {
        return "Project{" +
                "id=" + id +
                ", title='" + title + '\'' +
                ", description='" + description + '\'' +
                ", channels=" + channels +
                ", ownerusername='" + ownerusername + '\'' +
                ", users=" + users +
                '}';
    }
}

File: SEP3-TIER1/BlazorTest/Model/Project.cs
using Newtonsoft.Json;
using System.Collections.Generic;

namespace BlazorTest.Model
{
    public class Project
    {
        [JsonProperty("id", NullValueHandling = NullValueHandling.Ignore)]
        public int Id { get; set; }

        [JsonProperty("title", NullValueHandling = NullValueHandling.Ignore)]
        public string Title { get; set; }

        [JsonProperty("description", NullValueHandling = NullValueHandling.Ignore)]
        public string Description { get; set; }

        [JsonProperty("channels", NullValueHandling = NullValueHandling.Ignore)]
        public ICollection<Channel> Channels { get; set; }

        [JsonProperty("ownerusername", NullValueHandling = NullValueHandling.Ignore)]
        public string OwnerUsername { get; set; }

        [JsonProperty("users", NullValueHandling = NullValueHandling.Ignore)]
        public virtual IEnumerable<ProjectUser> Users { get; set; }

    }
}